client/state/data-layer/wpcom/wordads/settings/index.js

```diff
@@ -124,9 +124,11 @@ export const handleSaveFailure = ( {
 	updateWordadsSettings( siteId, previousSettings ),
 	errorNotice(
 		error === 'invalid_paypal'
-			? translate( 'Please enter a valid PayPal email address.' )
+			? translate(
+					'Your account needs a valid PayPal email to receive earnings. Enter one below, then try saving again.'
```

Comment:
It would be better to check we have translation of the new copy 🙂

Reply:
You mean by using `i18n.fixMe`?